t/33_last_insert_id.t

```diff
@@ -0,0 +1,74 @@
+use strict;
```

Comment:
What is the difference between `t/31insertid.t` and `t/33_last_insert_id.t` tests? They looks like are same.

Reply:
One tests `$sth->{mysql_insertid}` and `$dbh->{mysql_insertid}` and the other tests `$dbh->last_insert_id(undef, undef, undef, undef)`

Reply:
Then this should be in one test file. Because t/33_last_insert_id.t is big copy+paste.